spi/common/verifiable-credentials-spi/src/main/java/org/eclipse/edc/iam/verifiablecredentials/spi/validation/TrustedIssuerRegistry.java

```diff
@@ -16,17 +16,38 @@
 
 import org.eclipse.edc.iam.verifiablecredentials.spi.model.Issuer;
 
-import java.util.Collection;
+import java.util.Set;
 
 /**
- * A list of trusted VC issuers
+ * Trusted issuer registry.
  */
 public interface TrustedIssuerRegistry {
 
-    void addIssuer(Issuer issuer);
+    String WILDCARD = "*";
+
+    /**
+     * Register a supported type for a trusted issuer.
+     *
+     * @param issuer the issuer
+     * @param type   the type
+     */
+    void register(Issuer issuer, String type);
+
+    /**
+     * Check if an issuer is trusted.
+     *
+     * @param issuer the issuer
+     * @return true if the issuer is trusted, false otherwise.
+     */
+    boolean isTrusted(Issuer issuer);
```

Comment:
is this really necessary? What is the use case of this method? couldn't someone just check if `getSupportedTypes(...).isEmpty()`?

Reply:
I prefer having that method. It makes it explicit, and technically there could be a trusted issuer whose types are not necessarily supported by the EDC deployment, although that would be a stretch.